scripts/linux_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash +x
+
+BINARY_URL="https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux"
+SCRIPT="Linux"
+
+green () { echo -e "\033[01;32m$1:\033[0m $2"; }
+
+warn () { echo -e "\033[01;33m[$SCRIPT]WARNING:\033[0m $1"; }
+
+fatal () { echo -e "\033[01;31m[$SCRIPT]ERROR:\033[0m $1" >&2; exit 1; }
+
+architecture_version () {
+    case $(uname -m) in
+        x86_64) echo "amd64" ;;
+        # arm64)  echo "arm64" ;; ARM64 is unsupported on Linux for now
+        *)      fatal "$(uname -m) unsupported." ;;
+    esac
+}
+
+check_supported_distro () {
+    grep '^ID.*=.*ubuntu' /etc/os-release &> /dev/null || fatal "Aborting: You are using an unsupported distribution."
+}
+
+install_thyra () {
+    arch="$(architecture_version)" || exit 1
+    curl -s -L "${BINARY_URL}_${arch}" -o thyra-server || fatal "binary download failed."
+
+    chmod +x thyra-server || fatal "change to executable failed."
+    (ls /usr/local/bin/ &> /dev/null || sudo mkdir -p /usr/local/bin) || fatal "/usr/local/bin creation failed."
```

Comment:
i think ```ls /usr/local/bin/ &> /dev/null``` is useless here

Reply:
and also i doubt there is a chance that /usr/local/bin does not exists. thats mean the whole line is useless for me

Reply:
Done here https://github.com/massalabs/thyra/pull/213/commits/2e613911c5bc7d47423bbf1c16fda6f84f4ff505